### DISCUSSION TITLE
Snapshot the Data Model

### DISCUSSION THREAD

**Discussion by pchebro** [directus/directus#11492] [Archive]
Hi, 
I'm following instructions in the below page to capture the snapshot and apply in different environment through CI/CD pipeline. 
https://docs.directus.io/reference/cli/#snapshot-the-data-model

But looks like the location where I kept snapshot.yaml is not being recognized. I have added snapshot.yaml file under root directory of my project and also tried inside ./extensions/migrations/snapshots/snapshot.yaml, 

Please advise where to place snapshot.yaml file and commit to git so it can be picked during the deployment to Cloud Foundry via CI/CD pipeline. 
I am using below command to apply snapshot.
npx directus schema apply --yes true ./extensions/migrations/snapshots/snapshot.yaml

or 

when I had file in root directory:
npx directus schema apply --yes true snapshot.yaml

Appreciate your prompt response. Thank you!

**Comment by iksent** [comment]
Try to omit "true" in your command:

`npx directus schema apply --yes ./path/to/snapshot.yaml`

We are also experienced a bug here.
